android/app/src/main/AndroidManifest.xml

```diff
@@ -42,7 +42,6 @@
                 <category android:name="android.intent.category.LAUNCHER"/>
             </intent-filter>
             <!-- App linking -->
-            <meta-data android:name="flutter_deeplinking_enabled" android:value="true" />
```

Comment:
Why are we removing this? We intentionally opted out of Flutter's default deep link handler: https://docs.flutter.dev/ui/navigation/deep-linking#migrating-from-plugin-based-deep-linking

Reply:
>We intentionally opted out of Flutter's default deep link handler

But this was opting it _in_, right?  And that's now the default, so there's no need to keep the flag.  We're not using a plugin-based deep link.  There's some description of the change towards the end of https://medium.com/flutter/whats-new-in-flutter-3-27-28341129570c.

Reply:
Derp, thanks. I wonder if the new plugin-based approach would've let us get the raw URL value, and let us solve our upstream issue we ended up working around with a different enrollment code scheme... https://github.com/flutter/flutter/issues/114630